save-backend/src/main/kotlin/com/saveourtool/save/backend/service/ProjectService.kt

```diff
@@ -118,9 +120,11 @@ class ProjectService(
     /**
      * @param name
      * @param organizationName
+     * @param statuses
      * @return project
      */
-    fun findByNameAndOrganizationName(name: String, organizationName: String) = projectRepository.findByNameAndOrganizationName(name, organizationName)
+    fun findByNameAndOrganizationNameAndStatusIn(name: String, organizationName: String, statuses: Set<ProjectStatus> = setOf(ProjectStatus.CREATED)) =
+            projectRepository.findByNameAndOrganizationName(name, organizationName)?.takeIf { it.status in statuses }
```

Comment:
it can be added on repository level: `findByNameAndOrganizationNameAndStatusIn`

Reply:
done